src/test/resources/alma/KUR00770801.json

```diff
@@ -93,6 +89,10 @@
     "license" : [ {
       "id" : "http://creativecommons.org/publicdomain/zero/1.0",
       "label" : "Creative Commons-Lizenz CC0 1.0 Universal"
-    } ]
+    } ],
+    "sourceOrganization" : {
+      "label" : "lobid Organisation",
+      "id" : "http://lobid.org/organisations/004250681-DUE01"
```

Comment:
These (here: `004250681-DUE01`) look like ALma-intenral identifiers. Is there a mapping to Sigel/ISILs?

Reply:
Did "grep 681-DUE01 " on a 500MB alma Marc export and it just revealed this one entry, so I assume it's not mapable. There is indeed [alma-institution-code-to-isil.xml](https://github.com/hbz/lobid-resources/blob/master/src/main/resources/alma/maps/alma-institution-code-to-isil.xml) but it has an entry like  `"a 6443" value="DE-61"` so this doesn't help here.

Reply:
Hm, I found that `004250681` does not only exist in `MNG  .g="004250681-DUE01"` but is also part of the identifier `MBD  .i="990042506810206443"` which ends in `6443` which maps to `DE-61`. This is strange  because `MBD` stands for `MemberDetails`. I would at least have expected the  `6443` part of `MNG  .g`. Will ask our Alma experts.

Reply:
Also note that `MBD`  will probably be no more mandatory, we can say this after testload2. If it's not mandatory we cannot rely on it, obviously.

Reply:
@blackwinter do you know more about how to get the ISIL from `MNG .g` resp. why the `6443` is missing there?

Reply:
Don't know about MNG, but as you already alluded to, we're [mapping](https://github.com/hbz/limetrans/blob/b31230f/src/main/resources/transformation/macros.xml#L83) the ISIL from the MMS ID's institution code.

Reply:
Also, `MNG$$g` is the "Originating system ID". It's the record's ID *in* the originating system, not the ID *of* the system.

Reply:
Hm, good pointer, if not entirely correct:  `002136771-HBZ01` is a mishmash of `aleph-id` and an identifier _of_ the originating system - at least of the name of the database (`HBZ01`), not necessarily the name/isil of the institution.
However, it's true that `MNG$$g` shouldn't be used to determine the originating system.

Reply:
Ah, right, totally forgot to go into that. The same ID is included in `035$$a` so I assume that it is in fact an ID of some system and that that system might be the SISIS importer (or something like that). It's true that it isn't - in its entirety - the ID from the actual SISIS system.